schema/snapcraft.yaml

```diff
@@ -288,6 +288,7 @@ properties:
             uniqueItems: true
             items:
               type: string
+              pattern: "(no-system-libraries|no-install)"
```

Comment:
As we add values here, it might be better to use an `enum` like on https://github.com/snapcore/snapcraft/pull/1432/commits/6394486535ef6ed6c7ba0d086bfad056870a77f5#diff-fa3e006298d99e14b7fb303c7883c895R252